src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -976,22 +962,23 @@ def customizePerJob(self, job):
             jdl.append('+DESIRED_CMSDataLocations = "%s"\n' % ','.join(job['inputDatasetLocations']))
 
         # HighIO jobs
-        highIOjob = job.get('highIOjob')
-        if highIOjob:
+        if job.get('highIOjob'):
             jdl.append('+Requestioslots = 1\n')
+        else:
+            jdl.append('+Requestioslots = 0\n')
 
         # Performance estimates
-        if job.get('estimatedJobTime'):
-            jdl.append('+MaxWallTimeMins = %d\n' % int(job['estimatedJobTime'] / 60.0))
         if job.get('estimatedMemoryUsage'):
```

Comment:
From your HN, condor sets it to 1250MB in case it's not provided (logcollect, cleanup, harvesting(?)).
How about we default it to 1000MB?

For `estimatedDiskUsage`, I'd expect worker nodes to provide *at least* 4 GB, how about we default to that number (I don't remember the unit here though...)